src/types.tsx

```diff
@@ -256,20 +262,30 @@ export interface ScreenProps extends ViewProps {
    * - "landscape_right" – landscape-right orientation is permitted
    */
   screenOrientation?: ScreenOrientationTypes;
+  screenStyle?: StyleProp<ViewStyle>;
```

Comment:
What does this prop do?

Reply:
Added comment with explanation & changed the type to `StyleProp<Pick<ViewStyle, 'backgroundColor'>>`.

The explanation looks like follows right now:

```
  /**
   * Allows to set background color for the `Screen` component itself.
   * This might come handy when using `formSheet` stack presentation, when the content view is clipped.
   */

```